src/js/components/ServerErrorModal.js

```diff
@@ -62,18 +63,34 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
     });
   }
 
+  getFooter() {
+    return (
+      <div className="button-collection text-align-center flush-bottom">
+        <div className="button" onClick={this.handleModalClose}>
+          Close
+        </div>
+      </div>
+    );
+  }
+
   getContent() {
-    let errors = this.state.errors.map(function (error, i) {
+    let {errors} = this.state;
+    let errorMessages = this.state.errors.map(function (error, i) {
+      let errorMessageClass = classNames('text-align-center', {
+        // Last error message doesn't have margin bottom.
+        'flush-bottom': i === errors.length - 1
```

Comment:
Could we create a variable for the length to not re-compute it on every iteration?